src/Routes/Sign-In/Sign-in.jsx

```diff
@@ -0,0 +1,18 @@
+import { Input } from "../../component"
+
+export function SignIn (){
+  return (
+    <section className='mt-8 m-auto md:max-w-[600px] md:mt-14 md:pb-5 md:rounded-3xl md:border md:border-solid md:border-slate-50  '>
+      <h2 className='text-center md:mt-5 md:text-xl'>Sign In </h2>
+      <form action='' className='flex flex-col'>
+        <Input name={'Email'} id={'email'} type={'email'} placeholder={'Write your email here!'}/>
+        <Input name={'Password'} id={'password'} type={'password'} placeholder={'Write your password here!'}/>
```

Comment:
Un detalle por las dudas!: No son necesarios los {} wrappeando los strings. Entonces podría quedar mas simple como:
```suggestion
        <Input name='Email' id='email' type='email' placeholder='Write your email here!' />
        <Input name='Password' id='password' type='password' placeholder='Write your password here!'/>
```

Son necesarios cuando hacemos referencia a una variable, por ejemplo si tuvieramos:
```
const name = 'email'
<Input name={name} .../>
```